test/exchange/ExchangeTests.js

```diff
@@ -132,7 +132,7 @@ describe('Exchange Contract', () => {
 
     it('Supports the Exchange Interface', async () => {
         // IExchange Interface
-        expect(await exchange.supportsInterface("0xdf858c9f")).to.equal(true);
+        expect(await exchange.supportsInterface("0x581b76ff")).to.equal(true);
```

Comment:
For my knowledge, how are you getting this value? Is this just a hash of all the function signatures into one string?

Reply:
I used console.log() to print out the "type(IExchange).interfaceId" and then copied it over to this test. 

you're correct, it's the hash of all function signatures of the contract